site2/website/release-notes.md

```diff
@@ -1,5 +1,134 @@
 
-## Apache
+## Apache Pulsar Release Notes
+
+### 2.5.0 &mdash; 2019-12-06 <a id="2.5.0"></a>
+
+#### PIPs
+
+- [PIP-41] Introduce Protocol Handler API [#5130](https://github.com/apache/pulsar/pull/5130) [#5743](https://github.com/apache/pulsar/pull/5473) 
+- [PIP-45] Pluggable metadata interface [#5330](https://github.com/apache/pulsar/pull/5330) 
+- [PIP-44] Separate schema compatibility checker for producer and consumer [#5227](https://github.com/apache/pulsar/pull/5227)
+- [PIP-43] Producer send messages with different schema [#5141](https://github.com/apache/pulsar/issues/5141) [#5517](https://github.com/apache/pulsar/pull/5517) 
+- [PIP-51] Introduce sticky consumer [#5388](https://github.com/apache/pulsar/pull/5388)
+- [PIP-38] Support batch receive in java client. [#4621](https://github.com/apache/pulsar/pull/4621)
+- [PIP-52] PIP-52: [pulsar-sever] Add support of dispatch throttling relative to publish-rate [#5797](https://github.com/apache/pulsar/pull/5797)
```

Comment:
```suggestion
- [PIP-52] [pulsar-sever] Add support of dispatch throttling relative to publish-rate [#5797](https://github.com/apache/pulsar/pull/5797)
```

One question for overall:

Do we need to add a component for each item? So that users know which component has been updated/modified at first sight.

For example, [Broker], [Pulsar Manager], [Function], [Connector]

If so, we need to add [component] for all sections.